static/js/modules/election-lookup.js

```diff
@@ -193,6 +194,24 @@ ElectionLookup.prototype.init = function() {
   });
 };
 
+ElectionLookup.prototype.getUpcomingElections = function() {
+  // Call the API to get a list of upcoming election dates
+  var now = new Date();
```

Comment:
I know at least in the CMS there's a few places we check for dates, do you think it would be helpful if we have a helper function that just returns current date and other common date functionality?